scripts/cmake/vcpkg_configure_meson.cmake

```
@@ -182,47 +184,47 @@ function(vcpkg_internal_meson_generate_native_file_config _config) #https://meso
         endif()
     endif()
 
-    string(APPEND NATIVE_${_config} "VCPKG_TARGET_TRIPLET = '${TARGET_TRIPLET}'\n")
-    string(APPEND NATIVE_${_config} "VCPKG_CHAINLOAD_TOOLCHAIN_FILE = '${VCPKG_CHAINLOAD_TOOLCHAIN_FILE}'\n")
-    string(APPEND NATIVE_${_config} "VCPKG_CRT_LINKAGE = '${VCPKG_CRT_LINKAGE}'\n")
+    string(APPEND native_${config_type} "VCPKG_TARGET_TRIPLET = '${TARGET_TRIPLET}'\n")
+    string(APPEND native_${config_type} "VCPKG_CHAINLOAD_TOOLCHAIN_FILE = '${VCPKG_CHAINLOAD_TOOLCHAIN_FILE}'\n")
+    string(APPEND native_${config_type} "VCPKG_CRT_LINKAGE = '${VCPKG_CRT_LINKAGE}'\n")
 
-    string(APPEND NATIVE_${_config} "[built-in options]\n")
+    string(APPEND native_${config_type} "[built-in options]\n")
     if(VCPKG_TARGET_IS_WINDOWS)
         if(VCPKG_CRT_LINKAGE STREQUAL "static")
-            set(CRT mt)
+            set(crt_type mt)
         else()
-            set(CRT md)
+            set(crt_type md)
         endif()
-        if(${_config} STREQUAL DEBUG)
-            set(CRT ${CRT}d)
+        if(${config_type} STREQUAL "DEBUG")
+            set(crt_type ${crt_type}d)
         endif()
-        string(APPEND NATIVE_${_config} "b_vscrt = '${CRT}'\n")
+        string(APPEND native_${config_type} "b_vscrt = '${crt_type}'\n")
     endif()
-    string(TOLOWER "${_config}" lowerconfig)
-    set(_file "${CURRENT_BUILDTREES_DIR}/meson-nativ-${TARGET_TRIPLET}-${lowerconfig}.log")
-    set(VCPKG_MESON_NATIVE_FILE_${_config} "${_file}" PARENT_SCOPE)
-    file(WRITE "${_file}" "${NATIVE_${_config}}")
+    string(TOLOWER "${config_type}" lowerconfig)
+    set(native_config_name "${CURRENT_BUILDTREES_DIR}/meson-nativ-${TARGET_TRIPLET}-${lowerconfig}.log")
```

```suggestion
    set(native_config_name "${CURRENT_BUILDTREES_DIR}/meson-native-${TARGET_TRIPLET}-${lowerconfig}.log")
```

typo in original code, might as well fix